### ISSUE TITLE
Inactive vs active project categorization

### ISSUE THREAD

**Issue by anaqreon** [redecentralize/alternative-internet#187]
A significant number of the listed projects appear to be inactive or dormant, based on indicators such as the last commit dates for their GitHub repos and updates of their web pages. It could be useful to start listing such projects under an "Inactive" or "Dormant" section. They often have great ideas and are worth investigating, but it's also disappointing to then discover that investing time in the project is not really an option without an active developer community.

**Comment by AaronM04** [comment]
That would be very useful. How do you propose keeping a project's Active/Inactive state up-to-date?

**Comment by anaqreon** [comment]
My guess is that projects that show no active development or updates for over six months are most likely not going to resume activity. Sad but often true. By leaving such projects in the list, however, an interested person might discover that the project has revived, and could make a pull request to move it out of the inactive category. 

I believe it's worth the occasional miscategorization to improve the relevance of the list. We want people to find projects that inspire them to contribute instead of wasting their time (and lowering morale) sifting through dead projects.

**Comment by millette** [comment]
One could also assume that a self-respecting redecentralized application will probably have its source elsewhere. As in, there's not just github out there.

But to keep it simple, showing that a project is "inactive on github" could have some value - but let's remember it's not telling the whole story either.

**Comment by anaqreon** [comment]
I agree. I used the GitHub repo status as one of several possible indicators. I'm assuming that there is some level of diligence here, because we typically don't *want* projects to be inactive.

As an example, consider Avatar (avatar.ai). The website is rich with detailed graphics and information, and it's an exciting project. Then you get to the contact page and see that the discussion links (https://discussions.avatar.ai/user/marko-poloj%C3%A4rvi) lead you to an Nginx error. I remember when these led you to a discussion forum of some kind. The Twitter link reveals an account whose last post was on 2015/03/15, while founder Marko Polojärvi and André Staltz have been tweeting frequently and continuously on their personal feeds. This is the kind of project I would list as inactive.

I suppose it would be even better to actually learn from the developers if the project is officially dead or dormant. Given the effort it takes to check a few links and such to determine the apparent status, sending a message to the listed project contacts would be even better, but shouldn't be required in order to confidently place a project in the inactive category.

**Comment by nodiscc** [comment]
Hi, we're having the very same discussion at https://github.com/Kickball/awesome-selfhosted/issues/63. Parsing RSS feeds was mentioned for automation (check the last date of github release feeds, project news feeds...).
